android/start.md

```diff
@@ -1,116 +1,277 @@
+    <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.READ_PHONE_STATE" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+
+            <!--other code-->
+
+        <application
+            android:allowBackup="true"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
+            android:roundIcon="@mipmap/ic_launcher_round"
+            android:supportsRtl="true"
+            android:theme="@style/AppTheme">
+
+            <service android:name="org.eclipse.paho.android.service.MqttService" />
+
+            <!--other code-->
+        </application>
+```
+
+**Build your Android Studio project**.
+
+## Step 2: Set Up Your Backend
+
+Create new AWS backend resources and pull the AWS services configuration into the app. In a terminal window, navigate to your project folder (the folder that typically contains your project level `build.gradle`), and run the following command (for this app, accepting all defaults is OK):
+
+```bash
+$ cd ./YOUR_PROJECT_FOLDER
+$ amplify init        #accept defaults
+$ amplify push        #creates configuration file
+```
+
+An `awsconfiguration.json` file will be created with your configuration and updated as features get added to your project by the Amplify CLI. The file is placed in the `./src/main/res/raw` directory of your Android Studio project and automatically used by the SDKs at runtime.
+
+## Step 3: How it Works
 
-* Add the following to your `AndroidManifest.xml` :
+Rather than configuring each service through a constructor or constants file, the AWS SDKs for Android support configuration through a centralized file called `awsconfiguration.json` which defines all the regions and service endpoints to communicate. Whenever you run `amplify push`, this file is automatically created allowing you to focus on your Swift application code. On Android projects the `awsconfiguration.json` will be placed into the `./src/main/res/raw` directory.
 
-  ```xml
-  <uses-permission android:name="android.permission.INTERNET"/>
-  <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
-  ```
+To verify that the CLI is set up for your app, run the following command.
 
-## Step 2: Install amplify
+```bash
+  $ amplify status
+  | Category | Resource name | Operation | Provider plugin |
+  | -------- | ------------- | --------- | --------------- |
+```
+
+The CLI displays a status table with no resources listed. As you add feature categories to your app and run `amplify push`, backend resources created for your app are listed in this table.
+
+## Step 4: Add API and Database
+
+Add a GraphQL API to your app and automatically provision a database with the following command (accepting all defaults is OK):
 
 ```bash
-$ npm install --save aws-amplify
+$ amplify add api     #select GraphQL
```

Comment:
Might mention that they should choose API key authentication for the examples below.